search/elasticsearch/util/src/java/org/sakaiproject/search/elasticsearch/BaseElasticSearchIndexBuilder.java

```diff
@@ -1050,14 +1036,14 @@ else if (propertyValue instanceof Collection)
                 values = (Collection<String>) propertyValue;
             else {
                 if (propertyValue != null)
-                    getLog().warn("Couldn't find what the value for '" + propertyName + "' was. It has been ignored. " + propertyName.getClass());
+                    log.warn("Couldn't find what the value for '{}' was. It has been ignored. {}", propertyName, propertyName.getClass());
                 values = Collections.emptyList();
             }
 
             //If this property was already present there (this shouldn't happen, but if it does everything must be stored
             if (properties.containsKey(propertyName)) {
-                getLog().warn("Two properties had a really similar name and were merged. This shouldn't happen! " + propertyName);
-                getLog().debug("Merged values '" + properties.get(propertyName) + "' with '" + values);
+                log.warn("Two properties had a really similar name and were merged. This shouldn't happen! " + propertyName);
+                log.debug("Merged values '" + properties.get(propertyName) + "' with '" + values);
```

Comment:
Use new logging format

Reply:
thanks Bernie for catching that, I missed that one!